src/contract_wrappers/modules/permission_manager/general_permission_manager_wrapper/index.ts

```diff
@@ -0,0 +1,13 @@
+/* istanbul ignore file */
+import { GeneralPermissionManager_3_0_0, isGeneralPermissionManager_3_0_0 } from './3.0.0';
+import { GeneralPermissionManager_3_1_0, isGeneralPermissionManager_3_1_0 } from './3.1.0';
+
+export type GeneralPermissionManager = GeneralPermissionManager_3_0_0 | GeneralPermissionManager_3_1_0;
+
+export {
+  GeneralPermissionManager_3_0_0,
+  isGeneralPermissionManager_3_0_0,
+  GeneralPermissionManager_3_1_0,
+  isGeneralPermissionManager_3_1_0,
+};
+export { default as GeneralPermissionManagerCommon } from './common';
```

Comment:
It should export `GeneralPermissionManagerTransactionParams` from `./common` also.

Reply:
There is other task regarding this. I proceed to close this issue.